web/themes/custom/nysenate_theme/src/patterns/components/nysenate-header/nysenate-header.twig

```diff
@@ -146,7 +146,10 @@
         </div>
       {% endif %}
     </div>
-    {% if type != 'senator' and error_page == false %}
+    {%
+      if not is_front and type != 'senator' and error_page == false
+      or is_front and homepage_hero_node_type in ['press_conference', 'public_hearing', 'session']
```

Comment:
This needs parenthesis to clarify the order of conjunctive logic.

Reply:
All set!